.github/workflows/build.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
         run: ./gradlew check assemble
       - name: publish snapshot
         run: ./gradlew publishToSonatype
+        run: ./gradlew publishMavenPublicationToSonatypeRepository closeSonatypeStagingRepository releaseSonatypeStagingRepository
```

Comment:
I think the line above does the same as `publishMavenPublicationToSonatypeRepository`, so this might cause that we upload the artifact twice. Also, for the other two tasks (closeSonatypeStagingRepository releaseSonatypeStagingRepository), [the plugin](https://github.com/gradle-nexus/publish-plugin) has the `closeAndReleaseSonatypeStagingRepository` one for when it's needed to run them both. I'm not sure if closing and releasing a staging repo is necessary when publishing a SNAPSHOT version though, as for the "legacy maven central" it wasn't the case.

Reply:
Oof, I mean, this line wasn't even supposed to be part of this PR, so good catch. Let me remove it!